spec/hashie/mash_spec.rb

```diff
@@ -503,6 +503,13 @@ class SubMash < Hashie::Mash
       expect(converted.to_hash['a'].first.is_a?(Hash)).to be_truthy
       expect(converted.to_hash['a'].first['c'].first.is_a?(Hashie::Mash)).to be_falsy
     end
+
+    it 'allows use of suffix in hash keys' do
+      initial_hash = { "#{SecureRandom.hex}_" => 'a' }
```

Comment:
I experienced very strange behavior using a static hash key in this test.

If I explicitly set ```initial_hash = { 'a_' => 'a' }``` than Hashie::Mash does not error out in Ruby 2.2.2. This is not consistent with the gems actual behavior when instantiating a new Mash object. This seems to be due to an issue with how RSpec is loaded, however I have not been able to track down the source of the unexpected behavior.